include/yaml-cpp/version.h

```diff
@@ -0,0 +1,17 @@
+#ifndef YAML_CPP_VERSION_H
```

Comment:
Please append _62B23520_7C8E_11DE_8A39_0800200C9A66 to this, which is a random nonce I generated for all such defines.